cli/defenseclaw/observability/v8_redaction_policy.py

```
@@ -0,0 +1,690 @@
+) -> tuple[V8YAMLMutation, ...]:
+    if name in BUILT_IN_PROFILES:
+        raise ValueError(f"built-in profile {name!r} cannot be edited")
+    observability = _observability(source)
+    profiles = observability.get("redaction_profiles")
+    existing = profiles.get(name) if isinstance(profiles, Mapping) else None
+    if existing is not None and not isinstance(existing, Mapping):
+        raise ValueError(f"custom profile {name!r} has an invalid source shape")
+    result = dict(existing or {})
+    if extends is not None:
+        if extends not in {"sensitive", "content", "strict"}:
+            raise ValueError("custom profiles must extend sensitive, content, or strict")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)